README.md

```diff
@@ -9,6 +9,16 @@ A wrapper of [Syncthing](https://github.com/syncthing/syncthing) for Android.
 
 [<img alt="Get it on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png" height="80">](https://play.google.com/store/apps/details?id=com.nutomic.syncthingandroid) [<img alt="Get it on F-Droid" src="https://f-droid.org/badge/get-it-on.png" height="80">](https://f-droid.org/app/com.nutomic.syncthingandroid)
 
+# Status: "Maintenance mode" - Co-maintainers welcome
+
+tl;dr: The app is still kept up to date, and contributions are still welcome - however even reviews for those can take a long time. Co-maintainers are very welcome - get in touch if you are interested.
+
+No-one is dedicating significant time into development or reviews. It's still kept up to date with syncthing, android and dependencies under the wider syncthing project umbrella on a best effort basis. Contributions are reviewed, however available time for that is scarce so it will take time. And obviously it depends both on the size/clarity of the change and (admittedly subjective) relevance of it - chance of successful and speedier reviews is higher if your change is targeted and small.
```

Comment:
```suggestion
No-one is dedicating significant time into development or reviews. It's still kept up to date with Syncthing, Android and dependencies under the wider Syncthing project umbrella on a best effort basis. Contributions are reviewed, however available time for that is scarce so it will take time. And obviously it depends both on the size/clarity of the change and (admittedly subjective) relevance of it - chance of successful and speedier reviews is higher if your change is targeted and small.
```